components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/ApplicationMgtUtil.java

```diff
@@ -969,4 +971,18 @@ public static String buildSPData(ServiceProvider serviceProvider) {
         return StringUtils.EMPTY;
     }
 
+    /**
+     * This method is to check Adaptive authentication is availability.
+     *
+     * @return AdaptiveAuthentication Available or not.
+     */
+    public static boolean isAdaptiveAuthenticationAvailable() {
```

Comment:
we already have this verified in the framework right ? can't we use it

Reply:
Using that method (FrameworkUtils.isAdaptiveAuthenticationAvailable()) causes a circular dependancy